lesson13/js/main.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+//Modal
+const modalContainer = document.getElementById('js-modalContainer');
+const modalBg = document.createElement('div');
+const modalWindow = document.createElement('div');
```

Comment:
I think it would be better to give it an `id` so that it is called where it is used instead of defining it globally.

Reply:
Thank you for review!!
I had no idea of ​​giving an ID.
I was always worried about the location of variables, so This solved it!

[b275180](https://github.com/nyattsu72/hanson-frontend-cource/pull/18/commits/b275180225b60f88cc65e52284acfe1bf748071d)